pygmt/tests/test_clib_put_vector.py

```diff
@@ -122,7 +122,7 @@ def test_put_vector_string_dtype():
             "2021-02-03T00:00:00",
             "2021-02-03T04:00:00",
             "2021-02-03T04:05:06",
-            f"{datetime.utcnow().strftime('%Y-%m-%d')}T04:50:06",
+            f"{datetime.datetime.now(tz=datetime.UTC).strftime('%Y-%m-%d')}T04:50:06",
```

Comment:
Right, so [`datetime.UTC`](https://docs.python.org/3.12/library/datetime.html#datetime.UTC) is only available with Python 3.11+, so this test fails on Python 3.9. Could just remove the timezone component I suppose?

Reply:
Ah nevermind, found out that `datetime.UTC` is just an alias for `datetime.timezone.utc`. Changed in 90a3a497816cf85109c0acbdacd26a510cd70b14.